File: ParteC#/ProyectoProgramacion3/ProyectoProgramacion3Model/Model/crm/Cliente.cs
namespace ProyectoProgramacion3Model.Model.crm;
public class Cliente{
    public int idCliente { get; set; }
    public string nombres { get; set; } = string.Empty;
    public string apellidos { get; set; } = string.Empty;
    public TipoDocumento tipoDocumento { get; set; }
    public string numeroDocumento { get; set; } = string.Empty;
    public string nacionalidad { get; set; } = string.Empty;
    public string correo { get; set; } = string.Empty;
    public DateOnly fechaRegistro { get; set; }
    public string numeroContacto { get; set; } = string.Empty;
    public DateOnly fechaNacimiento { get; set; }
    public Cliente(int idCliente, string nombres, string apellidos, TipoDocumento tipoDocumento, string numeroDocumento, string nacionalidad, string correo, DateOnly fechaRegistro, string numeroContacto, DateOnly fechaNacimiento)
    {
        this.idCliente = idCliente;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.tipoDocumento = tipoDocumento;
        this.numeroDocumento = numeroDocumento;
        this.nacionalidad = nacionalidad;
        this.correo = correo;
        this.fechaRegistro = fechaRegistro;
        this.numeroContacto = numeroContacto;
        this.fechaNacimiento = fechaNacimiento;
    }
    public Cliente()
    {
    }
}

File: Backend/ParteJAVA/ProyectoPro3Modelo/src/main/java/pe/edu/pucp/proyectopro3/modelo/crm/Cliente.java
package pe.edu.pucp.proyectopro3.modelo.crm;

import java.util.Date;

public class Cliente {
    private int idCliente;
    private String nombres;
    private String apellidos;
    private TipoDocumento tipoDocumento;
    private String numeroDocumento;
    private String correo;
    private String nacionalidad;
    private Date fechaRegistro;
    private String numeroContacto;
    private Date fechaNacimiento;
    private Boolean activo;

    public Cliente(int idCliente, String nombres, String apellidos, TipoDocumento tipoDocumento, String numeroDocumento, String correo, String nacionalidad, Date fechaRegistro, String numeroContacto, Date fechaNacimiento, Boolean activo) {
        this.idCliente = idCliente;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.tipoDocumento = tipoDocumento;
        this.numeroDocumento = numeroDocumento;
        this.correo = correo;
        this.nacionalidad = nacionalidad;
        this.fechaRegistro = fechaRegistro;
        this.numeroContacto = numeroContacto;
        this.fechaNacimiento = fechaNacimiento;
        this.activo = activo;
    }

    public Cliente() {

    }

    public int getIdCliente() {
        return idCliente;
    }

    public void setIdCliente(int idCliente) {
        this.idCliente = idCliente;
    }

    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public TipoDocumento getTipoDocumento() {
        return tipoDocumento;
    }

    public void setTipoDocumento(TipoDocumento tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    public String getNumeroDocumento() {
        return numeroDocumento;
    }

    public void setNumeroDocumento(String numeroDocumento) {
        this.numeroDocumento = numeroDocumento;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getNacionalidad() {
        return nacionalidad;
    }

    public void setNacionalidad(String nacionalidad) {
        this.nacionalidad = nacionalidad;
    }

    public Date getFechaRegistro() {
        return fechaRegistro;
    }

    public void setFechaRegistro(Date fechaRegistro) {
        this.fechaRegistro = fechaRegistro;
    }

    public String getNumeroContacto() {
        return numeroContacto;
    }

    public void setNumeroContacto(String numeroContacto) {
        this.numeroContacto = numeroContacto;
    }

    public Date getFechaNacimiento() {
        return fechaNacimiento;
    }

    public void setFechaNacimiento(Date fechaNacimiento) {
        this.fechaNacimiento = fechaNacimiento;
    }

    public boolean getActivo(){ return activo; }

    public void setActivo(boolean activo){ this.activo = activo; }
}
